tests/virt_autotest/virt_utils.pm

```diff
@@ -48,14 +48,26 @@ sub repl_repo_in_sourcefile {
     my $verorig = "source.http.sles-" . get_version_for_daily_build_guest . "-64";
     my $veritem = check_var('ARCH', 'x86_64') ? $verorig : get_required_var('ARCH') . ".$verorig";
     if (get_var("REPO_0")) {
-        my $location = &virt_autotest_base::execute_script_run("", "perl /usr/share/qa/tools/location_detect_impl.pl", 60);
-        $location =~ s/[\r\n]+$//;
+	my $location = "";
+	if (check_var('ARCH', 's390x')) {
+            my $location = type_string("perl /usr/share/qa/tools/location_detect_impl.pl\n");
```

Comment:
For s390, type_string will not return real location, then the sourcefile variable will be wrong?